lib/perl/Genome/Model/ClinSeq/Command/CreateMutationSpectrum-exome.t

```diff
@@ -14,27 +14,35 @@ BEGIN {
 
 use above "Genome";
 use Test::More tests => 11;  #One per 'ok', 'is', etc. statement below
-use Genome::Model::ClinSeq::Command::CreateMutationSpectrum;
 use Data::Dumper;
+use Genome::Model::ClinSeq::TestData;
+use Genome::Utility::Test;
+use Sub::Override;
 
-use_ok('Genome::Model::ClinSeq::Command::CreateMutationSpectrum') or die;
+my $pkg = 'Genome::Model::ClinSeq::Command::CreateMutationSpectrum';
+use_ok($pkg);
+
+use Genome::Model::Build::ReferenceSequence;
+my $override = Sub::Override->new(
+    'Genome::Model::Build::ReferenceSequence::full_consensus_path',
+    sub { return '/gscmnt/ams1102/info/model_data/2869585698/build106942997/all_sequences.fa'; }
```

Comment:
I generally think it is okay to request "well-known" reference sequences in tests by name (e.g. `Genome::Model::Build::ReferenceSequence->get(name => 'GRCh37-lite-build37');`).    They should be included in the test database as well.

Barring that, it might be okay to symlink to the production version in the test data.  If that file moves we've got bigger trouble 😄